aviary/utils/process_input_decks.py

```diff
@@ -52,13 +52,23 @@ def create_vehicle(vehicle_deck=''):
     aircraft_values = get_option_defaults(engine=False)
 
     # TODO temporary, needed until debug_mode retired in favor of new verbosity flag
+    # TODO remove all hardcoded GASP values here, find appropriate place for them
     aircraft_values.set_val('debug_mode', val=False)
+    aircraft_values.set_val('INGASP.JENGSZ', val=4)
+    aircraft_values.set_val('test_mode', val=False)
+    aircraft_values.set_val('use_surrogates', val=True)
+    aircraft_values.set_val('mass_defect', val=10000, units='lbm')
+    aircraft_values.set_val('problem_type', val=ProblemType.SIZING)
+    aircraft_values.set_val(Aircraft.Electrical.HAS_HYBRID_SYSTEM, val=False)
+    aircraft_values.set_val(
```

Comment:
I believe this line is be able to be removed now thanks to #36
See @crecine's comment [here](https://github.com/OpenMDAO/om-Aviary/pull/69#discussion_r1454271630)

Reply:
got it, removed!